src/sql/workbench/contrib/notebook/test/electron-browser/cell.test.ts

```diff
@@ -1024,4 +1024,20 @@ suite('Cell Model', function (): void {
 		assert(!isEditMode);
 	});
 
+	test('Should read connection name from notebook metadata and use its corresponding connection profile', async function () {
```

Comment:
You don't seem to be checking that it's using the actual connection profile?

Reply:
Ah yes I'll change the description.